src/components/PageHeader/_PageHeader.scss

```diff
@@ -99,3 +100,27 @@
     }
   }
 }
+
+.coa-PageHeader--search {
+  padding-top: $coa-spacing-large;
+  padding-bottom: $coa-space-level3;
+  margin-bottom: $coa-spacing-md-small;
+  background: $coa-color-secondary;
+  h1 {
+    padding-top: $coa-spacing-xsmall;
+    font-size: $coa-font-size-medium;
+    text-align: center;
+  }
+  @include media($coa-medium-screen) {
```

Comment:
`@include tablet {` should work for this line

Reply:
So, here what I really needed was the `@include mobile` size